src/traffic_cop/traffic_cop_util.cpp

```diff
@@ -51,11 +53,27 @@ std::unique_ptr<planner::AbstractPlanNode> TrafficCopUtil::Optimize(
       auto types = order_by->GetOrderByTypes();
       auto exprs = order_by->GetOrderByExpressions();
       for (size_t idx = 0; idx < order_by->GetOrderByExpressionsSize(); idx++) {
+        if (exprs[idx].Get()->GetExpressionType() == terrier::parser::ExpressionType::VALUE_CONSTANT) {
```

Comment:
1. This logic is somewhat complex and difficult to read. Please add documentation that explains what it does.
2. Add a comment that this should be moved into the binder or optimizer. TrafficCopUtil isn't the right place for this.

Reply:
Thank you, Professor Pavlo. I'd like to move this logic to the binder.